stream/clients/python/setup.py

```diff
@@ -19,7 +19,7 @@
 
 name = 'apache-bookkeeper-client'
 description = 'Apache BookKeeper client library'
-version = '4.12.0'
+version = '4.13.0'
```

Comment:
In my opinion you can push this change directly 

We are going to merge the release notes only when the VOTE has passed.

On the master branch you should commit 4.14.0 and on branch-4.13 you should commit 4.13.0

Reply:
we are going to commit this patch ONLY to master branch, and not to branch-4.13